src/cli.zig

```diff
@@ -1158,6 +1158,7 @@ pub const Command = struct {
             RootCommandMatcher.case("r"),
             RootCommandMatcher.case("remove"),
             RootCommandMatcher.case("rm"),
+            RootCommandMatcher.case("un"),
```

Comment:
Is "npm un" really a separate command or does npm just autocomplete the "un" to "uninstall" since no other command with the same prefix exists?

If it autocompletes, it would mean also "npm uni" or "npm unin" etc. are supported?

Reply:
hey its a seperate command marked as a proper alias [npm docs](https://docs.npmjs.com/cli/v7/commands/npm-uninstall), it works the same in both npm (& pnpm, the equivalent drop in replacement which does symlinking)

Reply:
> npm uninstall [<@scope>/]<pkg>...
> aliases: unlink, remove, rm, r, un

..perhaps add `unlink` while at it?

Reply:
I think bun unlink already exists that’s why I didn’t want to touch it

that may have broken a few things